client/src/Router.js

```diff
@@ -10,6 +10,7 @@ export default class Router extends Component {
         <Switch>
           <Route exact path="/" component={MealsListPage} />
```

Comment:
I've noticed you're adding a header and footer to each page. You can place these before and after your Switch.

Reply:
Added header to the Router file, footer has not been added as we only want it on some places.